server/src/main/java/org/apache/uniffle/server/ShuffleTaskManager.java

```diff
@@ -74,12 +74,7 @@ public class ShuffleTaskManager {
   // but when get blockId, performance will degrade a little which can be optimized by client configuration
   private Map<String, Map<Integer, Roaring64NavigableMap[]>> partitionsToBlockIds;
   private ShuffleBufferManager shuffleBufferManager;
-  private Map<String, Long> appIds = Maps.newConcurrentMap();
-  // appId -> shuffleId -> commit count
-  private Map<String, Map<Integer, AtomicInteger>> commitCounts = Maps.newConcurrentMap();
-  private Map<String, Map<Integer, Object>> commitLocks = Maps.newConcurrentMap();
-  // appId -> shuffleId -> blockIds
-  private Map<String, Map<Integer, Roaring64NavigableMap>> cachedBlockIds = Maps.newConcurrentMap();
+  private Map<String, ShuffleTaskInfo> shuffleTaskInfo = Maps.newConcurrentMap();
```

Comment:
`shuffleTaskInfo` -> `shuffleTaskInfos`?

Reply:
Maybe `ShuffleTasksInfo` could be better?

Reply:
> Maybe `ShuffleTasksInfo` could be better?

Why? Do you know this is a collection of `ShuffleTaskInfo` according to `ShuffleTaskInfo`?

Reply:
Fine, I got it.

Reply:
You misunderstand my meaning, I means that
```
 private Map<String, ShuffleTaskInfo> shuffleTaskInfos = Maps.newConcurrentMap();
```